dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -857,6 +857,14 @@ export const Explorer = ({
   };
 
   const getUpdatedDataConfig = (statsToken: statsChunk) => {
+    if (statsToken === null) {
+      return {
+        [GROUPBY]: [],
+        [AGGREGATIONS]: [],
+        span: undefined,
```

Comment:
Is it necessary to give span a value as undefined? Can't we assign {} ?

Reply:
done